src/GafferSceneUIModule/RenderPassEditorBinding.cpp

```diff
@@ -485,7 +486,12 @@ class OptionInspectorColumn : public PathColumn
 			result.icon = runTimeCast<const Color3fData>( inspectorResult->value() );
 			result.background = g_sourceTypeColors.at( (int)inspectorResult->sourceType() );
 			std::string toolTip;
-			if( const auto source = inspectorResult->source() )
+			if( inspectorResult->sourceType() == Inspector::Result::SourceType::Fallback )
+			{
+				toolTip = "Default value";
```

Comment:
Do you think there's any merit in prefixing this with "Source :", to tie it more closely to the same bit of the regular tooltip?

Reply:
I went back and forth on this one originally, but including "Source :" here does keep the tooltips more consistent and easier to scan when moving the mouse cursor across cells representing default and non-default values, so you've convinced me to add it back in 62a292c